tests/authentication_tests/test_interactive_authentication.py

```diff
@@ -37,12 +38,22 @@
     user_impl,
     site_info,
 )
+from tank.authentication.utils import sanitize_http_proxy
+from tank.authentication.sso_saml2.core.sso_saml2_core import (
+    SsoSaml2Core,
+    get_renew_path,
+)
+from tank.authentication.sso_saml2.core.authentication_session_data import (
+    AuthenticationSessionData,
+)
 
 import tank
 import tank_vendor.shotgun_api3
+from unittest import skipIf
 
 
-@skip_if_pyside_missing
+# @skip_if_pyside_missing
```

Comment:
Is it necessary this commented line?  I mean, If you are using another decorator , you can delete the commented line

Reply:
Good catch, I forgot this line while debugging. Thanks!